changelogs/fragments/79734-ansible-test-change-detection.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "ansible-test change detection - when using ``--changed`` outside of AZP, use ``git merge-base <branch> HEAD`` instead of ``git merge-base --fork-point <branch>`` (https://github.com/ansible/ansible/pull/79734)."
```

Comment:
```suggestion
  - "ansible-test local change detection - use ``git merge-base <branch> HEAD`` instead of ``git merge-base --fork-point <branch>`` (https://github.com/ansible/ansible/pull/79734)."
```